File: FCLCore/src/main/java/com/tungsten/fclcore/mod/modinfo/ForgeOldModMetadataLst.java
package com.tungsten.fclcore.mod.modinfo;

import java.util.List;

/// @author Glavo
public class ForgeOldModMetadataLst {
    private int modListVersion;
    private List<ForgeOldModMetadata> modList;

    public int getModListVersion() {
        return modListVersion;
    }

    public void setModListVersion(int modListVersion) {
        this.modListVersion = modListVersion;
    }

    public List<ForgeOldModMetadata> getModList() {
        return modList;
    }

    public void setModList(List<ForgeOldModMetadata> modList) {
        this.modList = modList;
    }
}

File: FCL/src/main/java/com/tungsten/fclcore/mod/modinfo/ForgeOldModMetadataLst.kt
@file:OptIn(kotlinx.serialization.InternalSerializationApi::class)

package com.tungsten.fclcore.mod.modinfo

import kotlinx.serialization.Serializable

@Serializable
data class ForgeOldModMetadataLst(
    val modListVersion: Int = 0,
    val modList: List<ForgeOldModMetadata> = emptyList()
)
